python_wrapper/susi_python/main.py

```diff
@@ -160,10 +160,14 @@ def add_device(access_token):
         session = device_info['session'] # session info
         identity = session['identity']
         name = identity['name']
+        curr_location = geocoder.ip('me')
         params2 = {
         'macid': macid,
         'name': name,
-        'device': 'Smart Speaker',
+        'room': str(room_name),
+        'device': str(speaker_name),
```

Comment:
If I'm understanding correctly, you're passing 6 parameters in the API call. There's no `device` query parameter anymore in the `/aaa/addNewDevice.json` endpoint. I think it should be removed. Please correct me if I'm wrong though. Thanks :-)

Reply:
Okay @Akshat-Jain , I'll update it accordingly. Thanks